cheeseshop/templates/csgo_position_heatmap.html

```diff
@@ -0,0 +1,41 @@
+{% extends "base.html" %}
+{% block title %}CS:GO Death Log{% endblock %}
+{% block head %}
+    <script src="https://code.jquery.com/jquery-1.11.1.js"></script>
+    <script src="/static/js/simpleheat.js"></script>
+    <script src="/static/js/de_heatmp.js"> </script>
```

Comment:
Wouldn't this need to be in the pr for it to work?

Reply:
There is a note in the commit message, I'm not sure how we want to bring in the de_heatmp stuff. Maybe at build time?

Reply:
Yeah. Cloning and installing from master at build time seems like the right way to go?

Reply:
That sounds fine to me, can we add the stuff to clone as part of the build to this PR then?